packages/quantification/src/validations.ts

```diff
@@ -0,0 +1,83 @@
+import type { Output } from '@nori-dot-com/ggit';
+
+import { SOIL_METRICS_RESULTS_NAME_SUFFIX } from './constants';
+
+/**
+ * Validates for baseline and future scenarios:
+ *
+ * 1. Each model run has at least three scenarios with the expected Soil Metrics result suffix
+ * 2. Each result scenario has a MapUnit property
+ * 3. Each result scenario MapUnit InputCrop has at least 10 years of data
+ * 4. Each result scenario MapUnit has a somsc property
+ * 5. Each result scenario MapUnit somsc property has at least 11 years of data
+ *
+ * Eleven years of data are needed to calculate a 10 year-over-year average
+ *
+ * @throws An error if any validation fails
+ */
+export const validateParsedModelRunsData = ({
```

Comment:
Not/future request: we should strive to handle validation using something like yup (we currently use this) or zod(arguably a 
 successor to yup as it allows validating the whole result object at once. Validating like this (iteratively and without a validation schema) doesn't scale and can create a lot of cruft

Some examples of schema validation can be found in the project package in this repo, the [migration cli tool in the monorepo](https://github.com/nori-dot-eco/nori/blob/db189d95746a9744cbc13a24f04efa50b1c60c22/nori-cli/src/cmd/tools/registry-migrations/prepare-removal-migration.ts#L60), and well as the [grant task in the contracts repo](https://github.com/nori-dot-eco/contracts/blob/0b859b4f5712525a02decb9ee05bdcc6a628f056/tasks/vesting.ts#L120)